### PR TITLE
fix title and some small issues in the hub preview modal

### LINKED ISSUE #811: Spin Up Hub has no title
The title bar just shows the URL (Windows 10, MS Edge):

image

### PR DESCRIPTION
**Pull request by karthik2804**
This PR fixes the title not being defined for spin-up hub. It also includes some minor clean-ups in the preview modal

- The timestamps are now human-readable
- The source and author fields are links and navigatable.  

It also adds spin up hub to the hamburger menu in the homepage for mobile devices.

fixes #811

### CODE REVIEW

File: spin-up-hub/dist/index.html [hunk 1]
```diff
@@ -40,6 +40,8 @@
       rel="stylesheet"
     />
 
+    <title>Spin up Hub</title>
```
Comment (itowlson) [inline spin-up-hub/dist/index.html:43]: I _think_ we usually refer to this with title-case

```suggestion
    <title>Spin Up Hub</title>
```

File: spin-up-hub/index.html [hunk 2]
```diff
@@ -40,6 +40,8 @@
       rel="stylesheet"
     />
 
+    <title>Spin up Hub</title>
```
Comment (itowlson) [inline spin-up-hub/index.html:43]: Oh wait was that the compiled version

```suggestion
    <title>Spin Up Hub</title>
```

File: spin-up-hub/src/components/PreviewModal.vue [hunk 3]
```diff
@@ -62,11 +62,11 @@ export default {
                             </a>
                             <div class="metadata">
                                 <div class="name">Url</div>
-                                <div class="value">{{ modalData.url }}</div>
+                                <div class="value"><a href="{{ modalData.url }}">{{ modalData.url }}</a></div>
                             </div>
                             <div class="metadata">
                                 <div class="name">Submitted by</div>
-                                <div class="value">@{{ modalData.author }}
+                                <div class="value"><a href="https://github.com/{{ modalData.author }}">@{{ modalData.author }}</a>
```
Comment (itowlson) [inline spin-up-hub/src/components/PreviewModal.vue:69]: I am dubious about sending people to the author's GH page - few people make this a welcoming place.  Send them to the repo or to a user-provided homepage.

(Indeed a bunch of our stuff has the author as `fermyon`, and github.com/fermyon is completely un-useful.)

**Review by itowlson — CHANGES_REQUESTED** [review]

File: spin-up-hub/src/components/PreviewModal.vue (hunk 3 above)
Comment (karthik2804) [reply spin-up-hub/src/components/PreviewModal.vue:69]: We already have links sending them to the repo. I am happy to make it a non-link again. I do not have any strong feelings about letting people submit their own homepages but that would require some more refactoring so not a part of this PR.

Let me know if you think just returning it to being non-links would be sufficient for the time being. I do recall conversations with people asking for it to be linked to people's profiles as a sign of recognition.

Comment (itowlson) [reply spin-up-hub/src/components/PreviewModal.vue:69]: Nah, the only real blocker was the capitalisation of the title. :shipit:

File: spin-up-hub/index.html (hunk 2 above)
Comment (karthik2804) [reply spin-up-hub/index.html]: fixed!

**Review by itowlson — APPROVED** [review]

**Pull request merged**